### PR TITLE
Added Brazilian Portuguese Translation

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

**Comment by JoniVR** [comment]
I think you missed one or two strings, everything else looks good! Thanks!

image

I also added a commit to fix a truncation issue, this way the user can resize the table columns themselves if something is getting truncated.

**Comment by jpereira** [comment]
I didn't find that point. even because the next "Advanced" is already translated.

e.g:

```
[jpereira@sugarloaf:MonitorControl.git]$ grep Advanced -A1 -r $(find MonitorControl -iname pt-BR*)
MonitorControl/UI/pt-BR.lproj/Main.strings:/* Class = "NSTextFieldCell"; title = "Advanced"; ObjectID = "5wk-Dy-0fG"; */
MonitorControl/UI/pt-BR.lproj/Main.strings-"5wk-Dy-0fG.title" = "Avançado";
[jpereira@sugarloaf:MonitorControl.git]$
```

File: MonitorControl/Support/pt-BR.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+/* Sown in menu */
```
Comment (h3nnn4n) [inline MonitorControl/Support/pt-BR.lproj/Localizable.strings:2]: Typo

File: MonitorControl/UI/pt-BR.lproj/Main.strings [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+"DGv-iu-Jg1.title" = "Table View Cell";
+
+/* Class = "NSTextFieldCell"; title = "Keys"; ObjectID = "Dcz-GG-1li"; */
+"Dcz-GG-1li.title" = "Teclas";
+
+/* Class = "NSTextFieldCell"; title = "General"; ObjectID = "ENU-js-huy"; */
+"ENU-js-huy.title" = "Geral";
+
+/* Class = "NSMenuItem"; title = "Minimal"; ObjectID = "Eq3-z9-yIo"; */
+"Eq3-z9-yIo.title" = "Minimo";
```
Comment (h3nnn4n) [inline MonitorControl/UI/pt-BR.lproj/Main.strings:39]: Typo. It is `Mínimo`

File: MonitorControl/UI/pt-BR.lproj/Main.strings [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+"cnb-Li-1lE.title" = "Table View Cell";
+
+/* Class = "NSTableColumn"; headerCell.title = "Display Name"; ObjectID = "dNl-I0-hcg"; */
+"dNl-I0-hcg.headerCell.title" = "Nome do Monitor";
+
+/* Class = "NSTableColumn"; headerCell.title = "ID"; ObjectID = "dgp-q7-cBK"; */
+"dgp-q7-cBK.headerCell.title" = "ID";
+
+/* Class = "NSButtonCell"; title = "Lower Contrast after Brightness"; ObjectID = "fhy-Er-0aI"; */
+"fhy-Er-0aI.title" = "Menor contraste após brilho";
```
Comment (h3nnn4n) [inline MonitorControl/UI/pt-BR.lproj/Main.strings:114]: "Diminuir contraste após brilho" might be a better translation. or maybe "Diminuir contraste junto com brilho". This is a hard one 🤔

**Comment by JoniVR** [comment]
@jpereira I have updated master with a fix for the "Advanced" string not being found, if you'd be willing to update everything one more time, it should probably be solved now 🙂

**Comment by jpereira** [comment]
Hi @JoniVR , for sure. I updated the branch right now. thanks, @h3nnn4n for the review.

**Comment by JoniVR** [comment]
Hi @jpereira, thanks for updating so swiftly, I noticed you were still missing some translations. I should probably have mentioned that it would be easier to just merge in master. 

I just did it for you and resolved the conflicts, but I can't seem to push them to this Pull Request, do you have the `Allow edits from maintainers` toggle on the right side checked?

image

If you'd rather do it yourself, here's the steps you need to take:

1. Merge master branch into this PR branch.
2. run `bartycrouch update` inside the root directory of this project.
3. Update the missing translations.

**Comment by jpereira** [comment]
It is already enabled.
image

**Comment by jpereira** [comment]
@JoniVR I updated as you suggested.

**Comment by h3nnn4n** [comment]
Bump?

**Pull request closed without merging**